README.md

```diff
@@ -44,6 +44,18 @@ To get a local copy up and running follow these simple example steps.
 - Twitter: [@twitterhandle](https://twitter.com/@ememcookey)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/emem-ekpo-857135234/)
 
+**Author2**
+
+- GitHub: [@githubhandle](https://github.com/ecekpo)
+- Twitter: [@twitterhandle](https://twitter.com/@ememcookey)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emem-ekpo-857135234/)
+
+**Author3**
+
+- GitHub: [@githubhandle](https://github.com/ecekpo)
+- Twitter: [@twitterhandle](https://twitter.com/@ememcookey)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emem-ekpo-857135234/)
```

Comment:
[OPTIONAL]

Consider removing unnecessary fields on your README file, remember that this file is often your first chance to make you project look professional :briefcase: